roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
+  changed_when: false
+  register: vdsm_version
+
+- block:
+    - name: Change default timeout if setup
+      ini_file:
+        state: present
+        section: null
+        path: /etc/sysconfig/libvirt-guests
+        option: "SHUTDOWN_TIMEOUT"
+        value: "{{ host_deploy_shutdown_timeout }}"
+      when:
+        - host_deploy_shutdown_timeout is defined
+
+    - name: Change run in parallel if setup
+      ini_file:
+        state: present
+        section: null
+        path: /etc/sysconfig/libvirt-guests
+        option: "PARALLEL_SHUTDOWN "
+        value: "{{ host_deploy_shutdown_parallel }}"
+      when:
+        - host_deploy_shutdown_parallel is defined
```

Comment:
So whatabout the defaults, you,ve mentioned in readme?

Reply:
As already mentioned.